Helper/Cart.php

```diff
@@ -1353,6 +1355,7 @@ public function getCartData($paymentOnly, $placeOrderPayload, $immutableQuote =
      * @param $amount
      * @return  int
      */
+    // TODO remote this method
     public function getRoundAmount($amount)
```

Comment:
where is this used? if this is not used can we not get rid of it now?

Reply:
it's used in 10+ different location which is why I wanted to remove this in separate PR

Reply:
ahhh makes sense.